flow360/component/simulation/unit_system.py

```diff
@@ -35,6 +35,9 @@
 )
 u.dimensions.inverse_area = 1 / u.dimensions.area
 u.dimensions.inverse_length = 1 / u.dimensions.length
+u.dimensions.mass_flux = u.dimensions.mass / u.dimensions.time
+u.dimensions.velocity_squared = u.dimensions.length**2 * u.dimensions.time ** (-2)
```

Comment:
This is energy per mass. specific_energy?

Reply:
Done